templates/web/base/admin/contact-form.html

```diff
@@ -135,4 +135,13 @@
     <input type="hidden" name="token" value="[% csrf_token %]" >
     <input type="submit" class="btn" name="Create category" value="[% contact.in_storage ? loc('Save changes') : loc('Create category') %]" >
   </p>
+
+    <h2>[% loc('Extra data:') %] </h2>
+    <dl>
+      [% FOR pair IN contact.get_extra_metadata %]
+        <dt>[% pair.key %]</dt> <dd>[% pair.value %]</dd>
+      [% END %]
+    </dl>
+    [% INCLUDE 'admin/extra-metadata-form.html' metas=contact.get_metadata_for_input %]
```

Comment:
This needs to be `(contact.get_metadata_for_input OR [])` here for on a body page contact is undefined, this is an empty string, and merge creates a list of two entries.